### ISSUE TITLE
JavaScript error in tasks page

### ISSUE THREAD

**Issue by jaragunde** [Igalia/phpreport#399]
I'm getting this error on the JS console every time I load the tasks.php page:
```
TypeError: document.getElementById(...) is null
tasks.php:149:29
```

**Comment by jaragunde** [comment]
The error in Chromium looks like this:
```
Uncaught TypeError: Cannot read property 'scrollHeight' of null
    at tasks.php?date=2018-05-07:149
```
Also, it affects every page in the application; I think it's this piece of code in `header.php`:
```
        var HEADER_HEIGHT = document.getElementById('header').scrollHeight;
```

**Comment by jaragunde** [comment]
This is affecting #148.

**Comment by jaragunde** [comment]
```
commit 9ed6ea5f14bb5d46badd6fd31f30037e9754d4f3
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Tue May 29 19:48:21 2018 +0200

    [#399] Fix error when calculating HEADER_HEIGHT.
    
    As part of #323, the menubar div took the space of the header but the
    calculation of HEADER_HEIGHT was not modified. Later, the empty header
    was removed and the error started happening at
    25b50a45544d72e5d5ccfaa2239d56f7e1b462ef.
    
    Fixes regression on #148.
```
